bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/device/PJLinkDevice.java

```diff
@@ -324,9 +329,15 @@ public void setMute(MuteInstructionChannel channel, boolean muteOn)
     return new ErrorStatusQueryCommand(this).execute().getResult();
   }
 
-  public String getLampHours() throws ResponseException, IOException, AuthenticationException {
-    return new IdentificationCommand(this, IdentificationCommand.IdentificationProperty.LAMP_HOURS).execute()
-        .getResult();
+  public List<LampState> getLampStates() throws ResponseException, IOException, AuthenticationException {
+    return new LampStatesCommand(this).execute().getResult();
+  }
+
+  private CachedCommand<LampStatesResponse> cachedLampHoursCommand = new CachedCommand<LampStatesResponse>(
```

Comment:
Please  move this declaration to the top. It took me quite some time to find it while reviewing